app-api/src/main/java/org/openmrs/eip/app/CustomDatabaseOffsetBackingStore.java

```diff
@@ -0,0 +1,135 @@
+package org.openmrs.eip.app;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Map;
+
+import org.apache.commons.lang3.SerializationUtils;
+import org.apache.kafka.connect.runtime.WorkerConfig;
+import org.apache.kafka.connect.storage.Converter;
+import org.apache.kafka.connect.storage.MemoryOffsetBackingStore;
+import org.openmrs.eip.app.management.entity.DebeziumOffset;
+import org.openmrs.eip.app.management.repository.DebeziumOffsetRepository;
+import org.openmrs.eip.app.util.DebeziumOffsetUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import io.debezium.config.Instantiator;
+import io.debezium.embedded.EmbeddedEngine;
+
+@Component
+public class CustomDatabaseOffsetBackingStore extends MemoryOffsetBackingStore {
```

Comment:
In theory you are supposed to take CustomFileOffsetBackingStore and modify it to have this functionality in addition to its existing functionality and then rename it accordingly

Reply:
Done.

Reply:
Lets rename it before we merge this work otherwise I can't tell the diff

Reply:
I was thinking that we should use that name (from Custom**File**OffsetBackingStore to Custom**Database**OffsetBackingStore).
What is the name to use?

Reply:
What am saying is that don't rename the file for now, we will rename it later on, as a reviewer I just want to be able to view the changes in the backing store file otherwise github doesn't do a good job to render the diff when you rename and update a file, does this make sense?

Reply:
Yes, it makes sense. I have renamed it to the original name.